app/assets/javascripts/angular/controller/home-controller.js.erb

```diff
@@ -44,10 +66,15 @@ app.controller('HomeCtrl', ['$scope', '$http', '$rootScope', function($scope, $h
 
   $scope.$container.find('.item').each( function( i, itemElem ) {
     // make element draggable with Draggabilly
-    var draggie = new Draggabilly( itemElem );
+    // var draggie = new Draggabilly( itemElem );
     // bind Draggabilly events to Packery
-    $scope.$container.packery( 'bindDraggabillyEvents', draggie );
+    // $scope.$container.packery( 'bindDraggabillyEvents', draggie );
```

Comment:
there are layout issues I need to tackle next week. I may use the dragabilly stuff, I'm not sure yet. But for now it's distracting so I'm turning it off